### PR TITLE
Update Torq to v1.0.10

### PR DESCRIPTION
**Pull request by ohenrik**
Torq v1.0

New:
 - Connections status: Provides insight into the internals of Torq. (People often ask if there is a way to see if it's still importing data; well, now you can!)
 - LND: Select an outbound channel for payment
 - Keep historic information about tagging/untagging
 - Manually request a channel from an LSP that is following the LSP channel request specification (i.e. Deezy)
 - HTLC information in Open Channels
 - Quick filter with fuzzy search on listing screens
 - Mempool API integration for fee recommendations
 
New HTLC Firewall workflows:
 - You can create one or more workflows to firewall/drop HTLCs. (i.e. block any HTLC coming from peer1 going to peer2 below a certain amount. Or block any HTLC if the amount of pending HTLC for a peer is above a certain amount)
 - New HTLC Firewall Related workflow nodes:
   - HTLC Intercepted (Trigger): Every HTLC will trigger the workflow when HTLC interceptor is enabled in the node's settings
   - HTLC Event (Data Source): Starting point for the HTLC Firewall workflow
   - HTLC Filter (Filter): Allows you to filter base on the HTLC's incoming and/or outgoing amount
   - HTLC Channel Split (Filter): This splits the flow and you can chose to continue with either the incoming channel or outgoing channel from the HTLC
   - HTLC Channel Join (Filter): After a split you can chose to join the flow back together
   - HTLC Blocker (Action): This will firewall/drop the HTLC if something passed through it's parent

New workflow nodes:
- Close channel: be very careful with this one so you don't accidentally close a channel because of a logical error in your workflow!
- Open channel
- Request channel: You can now automatically request a channel from an LSP that is following the LSP channel request specification (i.e. Deezy)

Workflow update:
 - Dry-run a workflow
 - Capture workflow logs
 - Visualize workflow logs in a flow diagram, including very detailed information for debugging workflows
 - HTLC filterable data added to Channel Filter (i.e. summary of all pending HTLCs for peer1)
 
Updates:
 - Improved filtering on the peers page
 - Various optimizations and updates

Torq pay:
 - We've never hidden the fact that some day you will have to pay to use Torq.
 - This release starts a new phase for our startup.
 - You will need a subscription and we chose to use Stripe for the initial launch.
 - We ourselves are a startup looking for a good model to be able to provide the power of Torq to as many node operators as possible, so please don't hesitate to reach out to provide feedback! Let's take lightning to the next level, together!

### CODE REVIEW

**Review by nmfretz — COMMENTED** [review]
Thanks @ohenrik, this is a huge update! 

With so many updates since the last version on Umbrel (0.23.1), what would you like to see in the `releaseNotes` section of the umbrel-app.yml? We could include a few of the major, most relevant updates and then link to the release notes. Here's an example from btc-rpc-explorer:

https://github.com/getumbrel/umbrel-apps/blob/d7f186b0a59081d7fc00e859a1cfa65c7c51dbd1/btc-rpc-explorer/umbrel-app.yml#L18-L32

File: torq/docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 7028
 
   web:
-    image: lncapital/torq:0.23.1@sha256:6e968ccb3d6631e373d64d9853559a690f88e827f61b535b120f2620e003028f
+    image: lncapital/torq:1.0.10@sha256:4a9f469b40b27c56d4a0e65f706c79e7e053c39bb076e38250c50a873e395e84
```
[the remark this answers is not in the document]
Comment (ohenrik) [reply torq/docker-compose.yml:10]: Oh, I must have copied the wrong one.

[the remark this answers is not in the document]
Comment (ohenrik) [reply torq/docker-compose.yml]: Updated :)

**Comment by ohenrik** [comment]
> Thanks @ohenrik, this is a huge update!
> 
> With so many updates since the last version on Umbrel (0.23.1), what would you like to see in the `releaseNotes` section of the umbrel-app.yml? We could include a few of the major, most relevant updates and then link to the release notes. Here's an example from btc-rpc-explorer:
> 
> https://github.com/getumbrel/umbrel-apps/blob/d7f186b0a59081d7fc00e859a1cfa65c7c51dbd1/btc-rpc-explorer/umbrel-app.yml#L18-L32

Thank you! and good point, I'm a bit tired, so I missed adding it. Does the updated notes look good :)

**Comment by nmfretz** [comment]
Perfect, thanks @ohenrik! I have tested on both arm64 and x86(amd64). Going live now ⚡️

**Pull request merged**